fontbe/src/glyphs.rs

```diff
@@ -295,26 +295,15 @@ fn compute_deltas(
 
 /// convert raw deltas to the write-fonts representation (composite glyphs only)
 fn process_composite_deltas(deltas: Vec<Vec2>) -> Vec<GlyphDelta> {
-    let mut deltas = deltas
+    deltas
         .into_iter()
         .map(|delta| match delta.to_point().ot_round() {
             // IUP only applies to simple glyphs; for composites we
             // just mark the zero deltas as being interpolatable.
             (0, 0) => GlyphDelta::optional(0, 0),
             (x, y) => GlyphDelta::required(x, y),
         })
-        .collect::<Vec<_>>();
-
-    // match fonttools behaviour, ensuring there is at least one
-    // delta required, to ensure we write an entry for the glyf.
-    // <https://github.com/fonttools/fonttools/blob/0a3360e52727cdefce2e9b28286b074faf99033c/Lib/fontTools/varLib/__init__.py#L351>
-    // <https://github.com/fonttools/fonttools/issues/1381>
-    if deltas.iter().all(|delta| !delta.required) {
-        if let Some(first) = deltas.first_mut() {
-            first.required = true;
-        }
-    }
-    deltas
+        .collect::<Vec<_>>()
```

Comment:
optional but we no longer need to specify the type here, inference will figure out `collect()`.